content/source/docs/enterprise/private/reliability-availability.html.md

```diff
@@ -119,97 +119,58 @@ stores there.
 
-The maintainance of PostgreSQL and S3 are handled by the user, which includes backing
-up and restoring if necessary.
+### Demo
 
-_Configuration Data_ for the installation is stored in Docker volumes on the
-instance.
-
-The builtin Snapshot mechanism can be used to package up the _Configuration
-Data_ and store it off the instance. The builtin Restore mechanism can then be
-used to pull the configuration data back in and restore operations.  The
-Snapshot and Restore functionality is available to be configured and automated
-from the Admin Console.
-
-### Availability During Upgrades
-
-Upgrades for the Installer Architecture utilize the Installer Admin Console.
-Once an upgrade has been been detected (either online or airgap), the new code
-is imported and once ready, all services on the instance are restarted running
-the new code. The expected downtime is between 30 seconds and 5 minutes,
-depending on if database updates have to be applied.
-
-Only application services are changed during the upgrade; data is not backed up
-or restored. The only data changes that may occur during are the application of
-migrations the new version might apply to the _PostgreSQL Database_.
+All data (Configuration, PostgreSQL, Blob Storage, Vault) is stored within
+Docker volumes on the instance.
 
-When an upgrade is ready to start the new code, the system waits for all terrform runs
-to finish before continuing. Once the new code has started, the queue of runs is
-continued in the same order.
+**Snapshot:** The built-in Snapshot mechanism can be used to package up
+all data and store it off the instance. The frequency of automated snapshots can
+be configured hourly such that the worst-case data loss can be as low as 1 hour.
 
-### Recovery From Failures
+**Restore:** If the instance running Terraform Enterprise is lost, the only recovery
+mechanism in demo mode is to create a new instance and use the builtin Restore
+mechanism to recreate it from a previous snapshot.
 
-#### Demo mode
+[Configure Snapshot and Restore following these instructions](./automated-recovery.html).
 
-If the instance running Terraform Enterprise is lost, the only recovery mechanism
-in demo mode is to create a new instance and use the builtin Restore mechanism to
-recreate it from a previous snapshot.
+### Mounted Disk
 
-The frequency of automated snapshots can be configured such that worst-case
-data loss can be as low as 1 hour.
+_PostgreSQL Database_ and _Blob Storage_ use mounted disks for their
+data. Backup and restore of those volumes is the responsibility of the user, the
+system does not manage that.
 
-#### Mounted Disk
+_Configuration Data_ and _Vault Data_ for the installation is stored in Docker
```

Comment:
@evanphx wanted to validate this with you - is it accurate that the Vault data is stored in Docker volumes

Reply:
No, it's not correct. The vault key material is stored on the host in `/var/lib/tfe-vault` and the actual data that vault uses is in Postgres.